test/cjson_support.c

```diff
@@ -112,18 +112,22 @@ struct Interface *process_command(char *json_message, struct Interface *ifaces)
 			continue;
 		}
 
+		int is_delete = 0;
```

Comment:
pode colocar um comentário no início da função do porque teve que utilizar quase o mesmo process_command aqui no teste por favor. Pra quem pegar o código saber que não é uma réplica e do porque.

Reply:
Tranquilo, vou colocar